riff-raff/app/views/deploy/logSummary.scala.html

```diff
@@ -5,10 +5,10 @@
     }
     @defining(record.state) { state =>
         @state match {
-            case RunState.NotRunning => { <div class="alert alert-info"><p class="lead">Waiting to run</p></div> }
-            case RunState.Completed => { <div class="alert alert-success"><p class="lead">Completed</p></div> }
+            case RunState.NotRunning => { <div class="alert alert-info" data-run-state="NotRunning"><p class="lead">Waiting to run</p></div> }
```

Comment:
Won't the fact that the run-state is non-zero length disable the check whilst a deploy is waiting to run?